clients/services.ts

```diff
@@ -0,0 +1,35 @@
+import { agent } from "@pagopa/ts-commons";
+import {
+  AbortableFetch,
+  setFetchTimeout,
+  toFetch
+} from "@pagopa/ts-commons/lib/fetch";
+import { Millisecond } from "@pagopa/ts-commons/lib/units";
+import nodeFetch from "node-fetch";
+import { createClient } from "../generated/services-api/client";
+import { getConfigOrThrow } from "../utils/config";
+
+const config = getConfigOrThrow();
+
+export const servicesBaseUrl = config.SERVICES_API_URL;
+export const cgnSubscriptionKey = config.SERVICES_API_KEY;
```

Comment:
This variables are exported but are used locally. Maybe we can use directly `config.SERVICES_API_...` or use a local const not exported.